app/views/content/how-to-apply-for-teacher-training/teacher-training-personal-statement.md

```diff
@@ -40,73 +40,95 @@ quote:
     text: "Having a teacher training adviser was really beneficial when editing my personal statement and preparing for interviews. My top tips for the application process would be to get an adviser, and to think about what transferrable skills you have when writing your personal statement and answering interview questions."
     name: "Felix, former teacher trainee"
     classes: quote--indent
-
+  q-Josie: 
+    text: "Provided they have a good overall application and the right qualifications, most candidates who come to us for help with their personal statement will go on to get an interview." 
+    name: "Josie, teacher training adviser" 
+    classes: quote--indent 
 ---
 
 
-Your personal statement is your chance to make yourself memorable with teacher training providers and show them why you'll make a great teacher.
+A teacher training personal statement is a professional document that you’ll submit as part of your postgraduate teacher training application.  
+
+Your personal statement is your chance to make yourself memorable with teacher training providers.  
+
+You can use it to show:  
+
+* why you want to teach  
+
+* why you’ll make a great teacher  
+
+Writing a personal statement can feel daunting, but our teacher training advisers can help.  
```

Comment:
I like this but it feels a bit odd here as we don't tell people how to do this until the end of the page?

should this be consolidated in the TTA section later on (and the TTA section may be moved to earlier, above GSE, as it is more important)?